src/__mock__/wishlist.mock.ts

```diff
@@ -0,0 +1,56 @@
+import { AddWishlistCommand } from 'src/application/port/in/wishlist/WishlistUseCase';
+import { Wishlist, WishlistGroups, WishlistItem } from 'src/domain/wishlist';
+
+export const createWishlistMock: Readonly<AddWishlistCommand> = {
+  userId: 1,
+  productId: 101,
+  receiverName: '민수',
+};
+
+export const domainWishlistMock: Wishlist = {
+  id: 1,
+  userId: 1,
+  productId: 101,
+  receiverName: '민수',
+  createdAt: new Date('2025-05-21T10:00:00Z'),
+};
+
+export const validWishlistProduct = {
+  id: 101,
+  name: '감성 무드등',
+  imageUrl: 'https://example.com/image.jpg',
+  price: 10000,
+};
+
+export const wishlistGroupedMock: WishlistGroups[] = [
+  {
+    receiverName: '민수',
+    count: 1,
+    imageUrls: ['https://example.com/image.jpg'],
+  },
+];
+
+export const wishlistDetailMock: WishlistItem[] = [
+  {
+    wishlistId: 1,
+    product: {
+      id: 101,
+      name: '감성 무드등',
+      price: 10000,
+      imageUrl: 'https://example.com/image.jpg',
+    },
+    deleted: false,
+  },
+];
+export const wishlistDetailWithDeletedMock: WishlistItem[] = [
+  {
+    wishlistId: 2,
+    product: {
+      id: 999,
+      name: null,
+      price: null,
+      imageUrl: null,
+    },
+    deleted: true,
+  },
+];
```

Comment:
deleted는 사용하는 쪽에서 변환해서 사용해도 괜찮을 것 같아요

```suggestion
export const wishlistDetailMockData: WishlistItem = 
  {
    wishlistId: 1,
    product: wishlistProductMockData,
    deleted: false,
  };
```

Reply:
수정했습니다!